webpack.common.config.js

```diff
@@ -45,6 +45,7 @@ var entry = {
     'forgotpassword-page': staticPath('js/pages/forgotpassword-page.js'),
     'login-page': staticPath('js/pages/login-page.js'),
     'notifications-config-page': staticPath('js/pages/notifications-config-page.js'),
+    'share-embed-page': staticPath('js/pages/shareEmbedConfig.js'),
```

Comment:
Also, keep the name of the module consistent with the entry point name: `share-embed-page.js`.